kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KieServerMigrationTest.java

```diff
@@ -122,8 +126,12 @@ public void setup() {
     
     @After
     public void cleanup() {
-        kieServicesClient.disposeContainer(containerId);
-        kieServicesClient.disposeContainer(containerId2); 
+        if (kieServicesClient != null) {
+            ServiceResponse<Void> response = kieServicesClient.disposeContainer(containerId);
+            logger.info("Container {} disposed with response - {}", containerId, response.getMsg());
```

Comment:
As it's a disposal info, probably debug level is enough, wdyt?

Reply:
I left it as `info` on purpose as some of these failed tests are having problems when disposing the container, and in the current logs this information is not shown, so it's impossible to figure out why this is happening.